librz/include/rz_vector.h

```diff
@@ -140,7 +140,7 @@ RZ_API void *rz_vector_shrink(RzVector *vec);
 
 #define rz_vector_foreach_prev(vec, it) \
 	if (!rz_vector_empty (vec)) \
-		for (it = (void *)((char *)(vec)->a + (((vec)->len - 1)* (vec)->elem_size)); (char *)it != (char *)(vec)->a; it = (void *)((char *)it - (vec)->elem_size))
+		for (it = (void *)((char *)(vec)->a + (((vec)->len - 1)* (vec)->elem_size)); (char *)it != (char *)(vec)->a - (vec)->elem_size; it = (void *)((char *)it - (vec)->elem_size))
```

Comment:
@thestr4ng3r FYI

Reply:
Can you add a dedicated test for this in `test/unit/test_vector.c`?

Reply:
Done! ;)